department-of-veterans-affairs/abd-vro
# Hotfix/prodtest problems

## What was the problem?


Some 422 cases were doing offramp

Associated tickets or Slack threads:

- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How does this fix it?[^1]


None of the 422 will do offramp.

All 422 cases are now represented in the end to end test although "NOT" off-ramping is not checked. That is manually verified for app logs.

## How to test this PR
- Check the end to end test
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
